packages/react-devtools-shared/src/devtools/views/Components/Element.js

```diff
@@ -132,9 +132,10 @@ export default function Element({data, index, style}: Props) {
       className={className}
       onMouseEnter={handleMouseEnter}
       onMouseLeave={handleMouseLeave}
-      onMouseDown={handleMouseDown}
+      onClick={handleClick}
```

Comment:
This change made it so e2e tests could select a row (element) and then _click_ on it to inspect props.

Reply:
Can e2e tests not simulate mouse down?

Reply:
I can simulate a click by calling `element.click()` but to simulate a mouse-down I'd have to make a fake event and dispatch it etc. Seems dirtier :D